test/framework/utils/name.go

```diff
@@ -14,5 +14,5 @@ func RandomDNS1123Label(length int) string {
 
 	labelBuf := make([]byte, seedLen*2)
 	hex.Encode(labelBuf, seedBuf)
-	return string(labelBuf[:length])
```

Comment:
why do we change this RandomDNS1123Label? if we want a a prefix, we should use `fmt.Sprintf("a%v, RandomDNS1123Label(length-1))`

Reply:
RFC 1123 compliant labels cannot begin with a numeric character. Prepended "a" to force compliance.